packages/storybook/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/storybook",
-  "version": "0.5.1",
+  "version": "1.0.0-beta",
```

Comment:
Does this one actually need to be published?

Reply:
i will change the package to `private`. we should only publish package which has clear reason